legal-api/src/legal_api/utils/legislation_datetime.py

```diff
@@ -137,3 +137,17 @@ def is_future(date_string: str) -> bool:
         """Return the boolean for whether the date string is in the future."""
         effective_date = datetime.fromisoformat(date_string)
         return effective_date > datetime.utcnow().replace(tzinfo=timezone.utc)
+
+    @staticmethod
+    def get_start_of_date(date_string: str) -> datetime:
+        """Convert date string to date as UTC and return start of the day for a DateTime."""
+        date_utc = LegislationDatetime.as_utc_timezone_from_legislation_date_str(date_string)
+        start_of_day = datetime.combine(date_utc, datetime.min.time())
+        return start_of_day
+
+    @staticmethod
+    def get_end_of_date(date_string: str) -> datetime:
```

Comment:
can you move this funtion to https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/utils/datetime.py#L21 and rename to end_of_day

then you can use `datetime.end_of_day()`

Reply:
Should get_start_of_date() also be renamed and moved to datetime.py?

Reply:
I removed this function altogether and keep the code in query itself